### PR TITLE
Responses

### PR DESCRIPTION
**Pull request by Pouyan-Forghani**

### CODE REVIEW

File: app/views/surveys/index.html.erb [hunk 1]
```diff
@@ -19,6 +19,7 @@
         <td><%= survey.survey_category %></td>
         <td><%= "Incomplete" %></td>
         <td><%= link_to "Take Survey", select_language_survey_path(survey) %></td>
+        <td><%= link_to "See Your Responses", survey_responses_path(survey) %></td>
```
Comment (sidbav) [inline app/views/surveys/index.html.erb:22]: Instead of "See your Responses", maybe we say "See your latest response" or something along those lines?

**Review by sidbav — COMMENTED** [review]
Overall LGTM, we gotta have test coverage of these changes though! @fishercoburn10

**Review by sidbav — APPROVED** [review]

**Comment by sidbav** [comment]
Another comment:

We gotta pair these changes with the survey submission functionality, otherwise it will be quite confusing to make use of this feature!

**Review by chinmay10 — APPROVED** [review]
LGTM

**Pull request merged**